pulsar/client/client.py

```diff
@@ -342,7 +352,13 @@ def launch(self, command_line, dependencies_description=None, env=[], remote_sta
             job_config=job_config,
         )
         response = self.client_manager.exchange.publish("setup", launch_params)
-        log.info("Job published to setup message queue.")
+        log.info("Job published to setup message queue: %s", self.job_id)
+        return response
+
+    def get_status(self):
+        status_params = self._build_status_request_message()
+        response = self.client_manager.exchange.publish("setup", status_params)
```

Comment:
@jmchilton I think this should've been "status" since you created a new exchange for this?